polars/polars-core/src/series/implementations/fixed_size_list.rs

```diff
@@ -0,0 +1,232 @@
+use std::any::Any;
+use std::borrow::Cow;
+
+use super::{private, IntoSeries, SeriesTrait};
+use crate::chunked_array::comparison::*;
+use crate::chunked_array::ops::explode::ExplodeByOffsets;
+use crate::chunked_array::AsSinglePtr;
+use crate::fmt::FmtList;
+use crate::frame::groupby::*;
+use crate::prelude::*;
+use crate::series::implementations::SeriesWrap;
+use crate::series::IsSorted;
+
+impl private::PrivateSeries for SeriesWrap<FixedSizeListChunked> {
+    fn compute_len(&mut self) {
+        self.0.compute_len()
+    }
+    fn _field(&self) -> Cow<Field> {
+        Cow::Borrowed(self.0.ref_field())
+    }
+    fn _dtype(&self) -> &DataType {
+        self.0.ref_field().data_type()
+    }
+
+    fn _set_sorted_flag(&mut self, is_sorted: IsSorted) {
+        self.0.set_sorted_flag(is_sorted)
+    }
+
+    unsafe fn equal_element(&self, idx_self: usize, idx_other: usize, other: &Series) -> bool {
+        todo!();
```

Comment:
This file seems to have the bulk of the work left to do. It has a bunch of `todo!()` lines sprinkled throughout to get things to compile.

I copied this file from the list implementation and it's not clear which trait methods are crucial to have. For example the List implementation of `PrivateSeries` has `explode_by_offsets`, which doesn't make sense for a fixed size list. (Or maybe it does where the offset is always the constant size instead of a variable offset)

Any feedback/guidance on this file would be appreciated.

Reply:
Once I realized I needed to look at the source of the `ListChunked` impls (and not be distracted by the warnings about `PolarsNumericType`), it was pretty straightforward to move the todos out of _this file_.